src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -1136,16 +1148,16 @@ private void AddPatchNode(Patch patch, Vector2 position)
         nodes.Add(node.Patch, node);
     }
 
-    private Vector2 GetIndicatorPosition(Control parent, int dotIndex, Texture2D texture)
+    private Vector2 GetIndicatorPosition(PatchMapNode node, int dotIndex, Texture2D texture)
     {
         var indexModifier = MathF.Sin(dotIndex) * 0.5f + 0.5f;
-        var nodeModifier = parent.Position.LengthSquared();
-        var nodeCenter = parent.Position + new Vector2(Constants.PATCH_NODE_RECT_LENGTH / 2,
+        var nodeModifier = node.Patch.ID;
```

Comment:
Why does this use ID of the patch to modify the positions? It seems kind of suspicious to me. If in the future we have patch IDs like 4000-100000 range, would that cause issues? I really would not like to let any feature through that may complicate things if patch ID range is in the future expanded or dynamically generated or anything like that.